libraries/fabric-shim/cli.js

```diff
@@ -1,3 +1,9 @@
+/*
```

Comment:
As this is an executable bash script, the comments need to come after the `#!/usr/bin/env node`

Reply:
Nice catch!
Thanks for the careful review.